cli/pyproject.toml

```diff
@@ -24,7 +24,8 @@ prompt-toolkit = "^3.0.16"
 rich = "^13"
 python-dotenv = "^1.0.0"
 openpyxl = "^3.1.2"
+pywry = "^0.6.2"
```

Comment:
this will override the changes made in the pywry optional PR? not sure if you wanted it to happen this way

Reply:
All the random changed files like this are artifacts from the deltas between your branch and mine. If I point to develop, none of those files appear as changed.